pxr/imaging/garch/glPlatformDebugWindowWindows.cpp

```diff
@@ -165,8 +167,11 @@ Garch_GLPlatformDebugWindow::_MsgProc(HWND hWnd, UINT msg,
 {
     Garch_GLPlatformDebugWindow *window
         = Garch_GLPlatformDebugWindow::_GetWindowByHandle(hWnd);
+
+    // I'm not sure if the TF_VERIFY should actually be replace with a check against a nullptr, since
+    // this function is called during CreateWindowEx, before the WindowsMap is initialised
     if (!TF_VERIFY(window)) {
-        return 0;
+        return DefWindowProc(hWnd, msg, wParam, lParam);
```

Comment:
When the window is created, this function will be called already. Since the map used by _GetWindowByHandle() is only updated after window creation, it won't be able to find a valid *window pointer during creation.
In this case, I'm just passing the event to Windows to deal with it.
Previously, the plain return would eventually abort the program.